modules/KalturaSupport/resources/mw.KBaseMediaList.js

```diff
@@ -268,7 +268,7 @@
 		//Media Item
 		renderMediaList: function(callback){
 			//Only render if medialist item are present
-			if (this.getTemplateData().length > 0) {
+			if ( this.getTemplateData().length > 0 || ( this.getTemplateData().length === 0 && !this.getConfig('showEmptyPlaylistError') ) ) {
 				//Generate new list template data
```

Comment:
@eitanavgil , @vladm3  - I've changed the condition for better readability - please verify the logic works as it should after this change.